meson.build

```diff
@@ -312,6 +312,21 @@ foreach arg : ['-Wextra',
         endif
 endforeach
 
+want_sanitizer = get_option('sanitizer')
```

Comment:
`meson` allows passing [-Db_sanitize=...](http://mesonbuild.com/howtox.html#use-address-sanitizer). I'm not sure that another way of doing the same thing is needed.

Reply:
```
  b_sanitize  none          [none, address, thread, undefined, memory, address,undefined] Code sanitizer to use
```
This last commit seems to duplicate that.

Reply:
Indeed, I wasn't aware of that. I've removed the commit.